vdev/src/testing/integration.rs

```diff
@@ -84,6 +85,15 @@ impl IntegrationTest {
             args.push(filter.to_string());
         }
         args.extend(extra_args);
+
+        // Some arguments are not compatible with the --no-capture arg
+        let forbidden_args = vec!["--test-threads"];
+        if forbidden_args
+            .iter()
+            .any(|a| !args.contains(&(*a).to_string()))
```

Comment:
Since this is just a single argument to compare against, just do:
```suggestion
        if args.contains("--test-threads")
```
or, if the `.contains` doesn't work:
```suggestion
        if args.iter().any(|arg| arg == "--test-threads")
```